Services/PizzaService.cs

```diff
@@ -0,0 +1,97 @@
+using ContosoPizza.Models;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace ContosoPizza.Services
+{
+    /// <summary>
+    /// Pizza Service.
+    /// </summary>
+    public static class PizzaService
+    {
+        /// <summary>
+        /// Gets the pizzas.
+        /// </summary>
+        /// <value>
+        /// The pizzas.
+        /// </value>
+        static List<Pizza> Pizzas { get; }
+        /// <summary>
+        /// The next identifier
+        /// </summary>
+        static int nextId = 3;
+        /// <summary>
+        /// Initializes the <see cref="PizzaService"/> class.
+        /// </summary>
+        static PizzaService()
+        {
+            Pizzas = new List<Pizza>
+            {
+                new Pizza { Id = 1, Name = "Classic Italian", IsGlutenFree = false },
+                new Pizza { Id = 2, Name = "Veggie", IsGlutenFree = true }
+            };
+        }
+
+        /// <summary>
+        /// Gets all.
+        /// </summary>
+        /// <returns></returns>
+        public static List<Pizza> GetAll() => Pizzas;
+
+        /// <summary>
+        /// Gets the specified identifier.
+        /// </summary>
+        /// <param name="id">The identifier.</param>
+        /// <returns></returns>
+        public static Pizza Get(int id)
```

Comment:
This function will never null/exception, since it will return new default Pizza() at the end